airflow/models/renderedtifields.py

```diff
@@ -50,6 +50,12 @@
     from airflow.models.taskinstance import TaskInstance, TaskInstancePydantic
 
 
+def _get_fields(ti):
```

Comment:
This should have a better name to describe what the function does. `get_serialized_template_fields` maybe?